snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```diff
@@ -378,11 +378,46 @@ metrics:
           name: rttMonCtrlOperState
         table: rttMonCtrlOperTable
         tag: rtt_state
+        mapping:
+          1: reset
+          2: orderlyStop
+          3: immediateStop
+          4: pending
+          5: inactive
+          6: active
+          7: restart
```

Comment:
It seems that many mapping are camelCase instead of underscore_case. Since mapping will be used as tags, and tags are case insensitive it's preferred to use underscore_case.